commands/setup.go

```diff
@@ -61,6 +62,11 @@ func Setup() {
 		os.Exit(1)
 	}
 
+	if err := config.IsWritable(configFilepath); err != nil {
+		fmt.Printf("Config file %s is not writable: %s.\n", configFilepath, err)
```

Comment:
```suggestion
		fmt.Printf("Config file %s is not writable: %v.\n", configFilepath, err)
```

Reply:
Technically both correct, but golang team uses `%v` in their examples:
https://blog.golang.org/error-handling-and-go

Also in our codebase `%v` used for errors printing.